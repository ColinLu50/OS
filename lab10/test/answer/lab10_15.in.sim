24057
853
863
686
849
679
