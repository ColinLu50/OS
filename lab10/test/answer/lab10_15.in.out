FCFS
175 -> 381 -> 469 -> 46 -> 109 -> 56 -> 565 -> 515 -> 276 -> 224 -> 109 -> 539 -> 577 -> 603 -> 624 -> 475 -> 310 -> 547 -> 585 -> 254 -> 583 -> 541 -> 487 -> 224 -> 162 -> 277 -> 562 -> 660 -> 384 -> 73 -> 199 -> 581 -> 593 -> 688 -> 590 -> 151 -> 384 -> 136 -> 114 -> 487 -> 266 -> 302 -> 202 -> 619 -> 585 -> 357 -> 570 -> 66 -> 49 -> 50 -> 454 -> 177 -> 216 -> 401 -> 439 -> 154 -> 110 -> 154 -> 263 -> 417 -> 255 -> 520 -> 602 -> 441 -> 597 -> 244 -> 302 -> 353 -> 688 -> 342 -> 627 -> 34 -> 40 -> 267 -> 212 -> 637 -> 32 -> 329 -> 115 -> 321 -> 343 -> 668 -> 7 -> 224 -> 550 -> 209 -> 14 -> 322 -> 70 -> 385 -> 385 -> 596 -> 542 -> 465 -> 521 -> 285 -> 484 -> 234 -> 425 -> 423 -> 415 -> 257 -> 367 -> 608 -> 196 -> 311 -> 635 -> 372 -> 427 -> 376 -> 555 -> 418 -> 563 -> 628 -> 321 -> 640 -> 287 -> 197 -> 331 -> 570 -> 684 -> 243 -> 282 -> 414 -> 354
24057
SSTF
175 -> 177 -> 162 -> 154 -> 154 -> 151 -> 136 -> 115 -> 114 -> 110 -> 109 -> 109 -> 73 -> 70 -> 66 -> 56 -> 50 -> 49 -> 46 -> 40 -> 34 -> 32 -> 14 -> 7 -> 196 -> 197 -> 199 -> 202 -> 209 -> 212 -> 216 -> 224 -> 224 -> 224 -> 234 -> 243 -> 244 -> 254 -> 255 -> 257 -> 263 -> 266 -> 267 -> 276 -> 277 -> 282 -> 285 -> 287 -> 302 -> 302 -> 310 -> 311 -> 321 -> 321 -> 322 -> 329 -> 331 -> 342 -> 343 -> 353 -> 354 -> 357 -> 367 -> 372 -> 376 -> 381 -> 384 -> 384 -> 385 -> 385 -> 401 -> 414 -> 415 -> 417 -> 418 -> 423 -> 425 -> 427 -> 439 -> 441 -> 454 -> 465 -> 469 -> 475 -> 484 -> 487 -> 487 -> 515 -> 520 -> 521 -> 539 -> 541 -> 542 -> 547 -> 550 -> 555 -> 562 -> 563 -> 565 -> 570 -> 570 -> 577 -> 581 -> 583 -> 585 -> 585 -> 590 -> 593 -> 596 -> 597 -> 602 -> 603 -> 608 -> 619 -> 624 -> 627 -> 628 -> 635 -> 637 -> 640 -> 660 -> 668 -> 684 -> 688 -> 688
853
SCAN
175 -> 162 -> 154 -> 151 -> 136 -> 115 -> 114 -> 110 -> 109 -> 73 -> 70 -> 66 -> 56 -> 50 -> 49 -> 46 -> 40 -> 34 -> 32 -> 14 -> 7 -> 0 -> 177 -> 196 -> 197 -> 199 -> 202 -> 209 -> 212 -> 216 -> 224 -> 234 -> 243 -> 244 -> 254 -> 255 -> 257 -> 263 -> 266 -> 267 -> 276 -> 277 -> 282 -> 285 -> 287 -> 302 -> 310 -> 311 -> 321 -> 322 -> 329 -> 331 -> 342 -> 343 -> 353 -> 354 -> 357 -> 367 -> 372 -> 376 -> 381 -> 384 -> 385 -> 401 -> 414 -> 415 -> 417 -> 418 -> 423 -> 425 -> 427 -> 439 -> 441 -> 454 -> 465 -> 469 -> 475 -> 484 -> 487 -> 515 -> 520 -> 521 -> 539 -> 541 -> 542 -> 547 -> 550 -> 555 -> 562 -> 563 -> 565 -> 570 -> 577 -> 581 -> 583 -> 585 -> 590 -> 593 -> 596 -> 597 -> 602 -> 603 -> 608 -> 619 -> 624 -> 627 -> 628 -> 635 -> 637 -> 640 -> 660 -> 668 -> 684 -> 688
863
C-SCAN
175 -> 162 -> 154 -> 151 -> 136 -> 115 -> 114 -> 110 -> 109 -> 73 -> 70 -> 66 -> 56 -> 50 -> 49 -> 46 -> 40 -> 34 -> 32 -> 14 -> 7 -> 0 -> 688 -> 688 -> 684 -> 668 -> 660 -> 640 -> 637 -> 635 -> 628 -> 627 -> 624 -> 619 -> 608 -> 603 -> 602 -> 597 -> 596 -> 593 -> 590 -> 585 -> 583 -> 581 -> 577 -> 570 -> 565 -> 563 -> 562 -> 555 -> 550 -> 547 -> 542 -> 541 -> 539 -> 521 -> 520 -> 515 -> 487 -> 484 -> 475 -> 469 -> 465 -> 454 -> 441 -> 439 -> 427 -> 425 -> 423 -> 418 -> 417 -> 415 -> 414 -> 401 -> 385 -> 384 -> 381 -> 376 -> 372 -> 367 -> 357 -> 354 -> 353 -> 343 -> 342 -> 331 -> 329 -> 322 -> 321 -> 311 -> 310 -> 302 -> 287 -> 285 -> 282 -> 277 -> 276 -> 267 -> 266 -> 263 -> 257 -> 255 -> 254 -> 244 -> 243 -> 234 -> 224 -> 216 -> 212 -> 209 -> 202 -> 199 -> 197 -> 196 -> 177
686
LOOK
175 -> 162 -> 154 -> 151 -> 136 -> 115 -> 114 -> 110 -> 109 -> 73 -> 70 -> 66 -> 56 -> 50 -> 49 -> 46 -> 40 -> 34 -> 32 -> 14 -> 7 -> 177 -> 196 -> 197 -> 199 -> 202 -> 209 -> 212 -> 216 -> 224 -> 234 -> 243 -> 244 -> 254 -> 255 -> 257 -> 263 -> 266 -> 267 -> 276 -> 277 -> 282 -> 285 -> 287 -> 302 -> 310 -> 311 -> 321 -> 322 -> 329 -> 331 -> 342 -> 343 -> 353 -> 354 -> 357 -> 367 -> 372 -> 376 -> 381 -> 384 -> 385 -> 401 -> 414 -> 415 -> 417 -> 418 -> 423 -> 425 -> 427 -> 439 -> 441 -> 454 -> 465 -> 469 -> 475 -> 484 -> 487 -> 515 -> 520 -> 521 -> 539 -> 541 -> 542 -> 547 -> 550 -> 555 -> 562 -> 563 -> 565 -> 570 -> 577 -> 581 -> 583 -> 585 -> 590 -> 593 -> 596 -> 597 -> 602 -> 603 -> 608 -> 619 -> 624 -> 627 -> 628 -> 635 -> 637 -> 640 -> 660 -> 668 -> 684 -> 688
849
C-LOOK
175 -> 162 -> 154 -> 151 -> 136 -> 115 -> 114 -> 110 -> 109 -> 73 -> 70 -> 66 -> 56 -> 50 -> 49 -> 46 -> 40 -> 34 -> 32 -> 14 -> 7 -> 688 -> 684 -> 668 -> 660 -> 640 -> 637 -> 635 -> 628 -> 627 -> 624 -> 619 -> 608 -> 603 -> 602 -> 597 -> 596 -> 593 -> 590 -> 585 -> 583 -> 581 -> 577 -> 570 -> 565 -> 563 -> 562 -> 555 -> 550 -> 547 -> 542 -> 541 -> 539 -> 521 -> 520 -> 515 -> 487 -> 484 -> 475 -> 469 -> 465 -> 454 -> 441 -> 439 -> 427 -> 425 -> 423 -> 418 -> 417 -> 415 -> 414 -> 401 -> 385 -> 384 -> 381 -> 376 -> 372 -> 367 -> 357 -> 354 -> 353 -> 343 -> 342 -> 331 -> 329 -> 322 -> 321 -> 311 -> 310 -> 302 -> 287 -> 285 -> 282 -> 277 -> 276 -> 267 -> 266 -> 263 -> 257 -> 255 -> 254 -> 244 -> 243 -> 234 -> 224 -> 216 -> 212 -> 209 -> 202 -> 199 -> 197 -> 196 -> 177
679
